### PR TITLE
[fabric] move hashicorp vault functions to a single script

### LINKED ISSUE #2323: feat(shared): move hashicorp vault functions to a single script
As a developer, I want to improve code maintainability and efficiency by decoupling the vault related functionality from individual helm charts.

**Description:**

Currently, each helm chart contains its own vault related code responsible for various operations like validating vault responses, initializing vault tokens, reading vault secrets, and writing secrets to the vault. This code duplication across all Helm charts leads to inefficiencies and makes it challenging to maintain consistency.

---

**Solution:**

To address this issue, we will utilize the `bevel-vault.sh` script, provided as a ConfigMap by the `vault-script` Helm chart available in our shared directory. This script will enable us to perform vault related operations consistently across all helm charts.

---

**Additional information:**

The `vault-script` Helm chart simplifies interaction with the vault.

---

**The following platforms require fixes for this issue:**
- [x] Hyperledger Besu
- [x] GoQuorum
- [ ] Hyperledger-Fabric
- [ ] R3-Corda-Ent
- [ ] R3-Corda
- [ ] Substrate

### PR DESCRIPTION
**Pull request by saurabhkumarkardam**
### **Commit to be reviewed**
---

**feat(fabric): move hashicorp vault functions to a single script**

```
This commit decouples the vault related functionality from individual helm charts by utilizing the bevel-vault.sh script.

Changes made:

 • Removed vault-related code from individual Fabric Helm charts.
 • Updated all Helm charts to utilize the shared bevel-vault.sh script.
 • Deleted the vault_kubernetes_job.tpl template file from Fabric platform as we already move this feature to the shared platform.
 • Deleted the verify_chaincode Helm chart from Fabric platform as it is not in use and also there is no .tpl template file for the same Helm chart.

These changes will improve code maintainability and efficiency by reducing code duplication and making it easier to maintain consistency.
```

fixes #2323

### CODE REVIEW

File: platforms/hyperledger-fabric/charts/ca/templates/deployment.yaml [hunk 1]
```diff
@@ -61,14 +61,17 @@ spec:
             - key: fabric-ca-server-config.yaml
               path: fabric-ca-server-config.yaml
       {{- end }}        
-      {{ if .Values.vault.tls  }}
+      {{ if .Values.vault.tls  }} # true
```
Comment (sownak) [inline platforms/hyperledger-fabric/charts/ca/templates/deployment.yaml:64]: these comments are not right

**Review by sownak — CHANGES_REQUESTED** [review]
rest looks good

File: platforms/hyperledger-fabric/charts/ca/templates/deployment.yaml (hunk 1 above)
Comment (saurabhkumarkardam) [reply platforms/hyperledger-fabric/charts/ca/templates/deployment.yaml]: Corrections have been made. Thank you.

**Review by sownak — APPROVED** [review]

**Review by suvajit-sarkar — APPROVED** [review]
LGTM

**Pull request merged**